src/Services/TimeManager.vala

```diff
@@ -40,14 +43,22 @@ public class DateTime.Services.TimeManager : Gtk.Calendar {
 
         add_timeout ();
         try {
+            clockSettings = new GLib.Settings ("org.gnome.desktop.interface");
+            clockSettings.bind ("clock-show-seconds", this, "show-seconds", SettingsBindFlags.DEFAULT);
+
+            // Listen to Changes in the show-seconds property
+            this.notify["show-seconds"].connect ((sender, property) => {
```

Comment:
Remove `this` here. You also don't need the `sender, property` part, Vala doesn't enforce you to match number of parameters in lambda expressions.